graphql/mutations/share-with-a-friend-mutations.graphql

```diff
@@ -0,0 +1,4 @@
+extend type Mutation  @middleware(checks: ["auth:api"]) {
+    """ Send an invitation email """
+    ShareWithAFriend(name: String! phone_number: String email: String!): String @field(resolver: "ShareWithAFriend@resolve")
```

Comment:
@enwemasor1995 Remove the @field resolver, because you have changed resolve to __invoke above